src/components/pagination/custom-link-text/index.njk

```diff
@@ -0,0 +1,32 @@
+---
+title: Custom link text
```

Comment:
If we're not including this example in the guidance we can probs delete this page

Reply:
Good spot. Fixed!